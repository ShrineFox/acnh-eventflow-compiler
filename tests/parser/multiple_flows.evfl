flow Test1():
    TestActor2.Action0()


flow Test2():
    TestActor2.Action1(3)
    TestActor1.Action0()
flow Test3():
    return
