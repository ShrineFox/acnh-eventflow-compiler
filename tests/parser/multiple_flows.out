RootNode[name=Test1, vardefs=[], out_edges=[Event0]]
RootNode[name=Test2, vardefs=[], out_edges=[Event1]]
RootNode[name=Test3, vardefs=[], out_edges=[]]
ActionNode[name=Event0, action=EventFlowActionAction0(), params={}, out_edges=[]]
ActionNode[name=Event1, action=EventFlowActionAction1(param0), params={'param0': TypedValue(type=Type(type='int'), value=3)}, out_edges=[Event2]]
ActionNode[name=Event2, action=EventFlowActionAction0(), params={}, out_edges=[]]

